src/votekit/elections/election_types.py

```diff
@@ -900,3 +901,57 @@ def __init__(
         super().__init__(profile, ballot_ties)
         self.seats = seats
         self.tiebreak = tiebreak
+
+
+class HighestScore(Election):
+    """
+    Simulates an election based on points. Chooses the m candidates with highest scores.
+    Ties are broken by randomly permuting the tied candidates.
+
+    **Attributes**
+
+    `profile`
+    :   PreferenceProfile to run election on, with PointBallots.
+
+    `seats`
+    :   number of seats to be elected
+
+    `tiebreak`
+    : tiebreak method, defaults to random, supports 'none', 'random',
+
+    
+    """
+
+    def __init__(self, profile: PreferenceProfile, seats: int, tiebreak: str= "random"):
+        super().__init__(profile, ballot_ties = False)
+
+        self.seats = seats
+        self.tiebreak = tiebreak
+
+ 
+    def run_step(self):
+        # dictionary whose keys are candidates, values are vote totals
+        vote_tallies = self.state.profile.to_dict()
+
+        ranking = scores_into_set_list(score_dict = vote_tallies)
+        untied_ranking = tie_broken_ranking(ranking = ranking, profile = self.state.profile,
+                                     tiebreak=self.tiebreak)
+
+        if ranking != untied_ranking:
+            print("A tie was broken.")
```

Comment:
probably want to avoid `print` here, maybe `logging` or adding some other way of checking this after the fact if desired